changelogs/api-management-service.md

```diff
@@ -2,6 +2,28 @@
 
 Developer portal follows an independent release lifecycle and the [per-release changelog is available in the developer portal's GitHub repository](https://aka.ms/apimdevportal/releases).
 
+## Release - API Management service: January, 2022
+
+A regular Azure API Management service update was started on January 20, 2021, and included the following new features, bug fixes, and other improvements. It may take several weeks for your API Management service to receive the update.
+
+### Featured
+
+1. [Managed certificate support is now in public preview](https://azure.microsoft.com/updates/public-preview-managed-certificate-support-for-azure-api-management/).
+
+### New
+
```

Comment:
We added `East Asia` as a new supported region for Availability Zones https://aka.ms/apimaz. Do we announce those here?

Reply:
We also released new management sdk for .NET https://www.nuget.org/packages/Microsoft.Azure.Management.ApiManagement/8.0.0-preview

Reply:
Added